scalafix-tests/src/test/scala/scalafix/tests/GitHubUrlRewriteSuite.scala

```diff
@@ -20,4 +20,19 @@ class GitHubUrlRewriteSuite extends FunSuiteLike with Matchers {
         url.toString shouldBe "https://github.com/someorg/somerepo/blob/master~1/scalafix-rewrites/src/main/scala/somerepo/scalafix/Somerepo_1_2_3.scala"
     }
   }
+
+  test("it replaces invalid characters in package name with _") {
+    Conf.Str("github:someorg/some-repo/1.2.3") match {
+      case GitHubUrlRewrite(url) =>
+        url.toString shouldBe "https://github.com/someorg/some-repo/blob/master/scalafix-rewrites/src/main/scala/some_repo/scalafix/Some-repo_1_2_3.scala"
```

Comment:
Yeah, I noticed that too but I wasn't sure how to address it. Probably underscores are more consistent with the filename, so it makes sense.

Also, are you ok with keeping the capitalization? So it would be

```scala
s"${packageName.capitalize}_${normVersion}.scala"
```